ulauncher/ui/tray_icon.py

```diff
@@ -99,7 +100,11 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
 
         elif tray_icon_lib == "AyatanaIndicator":
             app_status = AyatanaIndicator.IndicatorCategory.APPLICATION_STATUS
-            self.aya_indicator = AyatanaIndicator.Indicator.new("ulauncher", icon_name, app_status)
+            self.aya_indicator = AyatanaIndicator.Indicator.new(
+                "ulauncher",
+                icon_name,
+                app_status,  # pyrefly: ignore[ bad-argument-type]
```

Comment:
Pyrefly detects that AyatanaIndicator can be one of two modules and that app_status can be a property of one of two modules, so it this is a false error because it's smart but not smart enough to understand AyatanaIndicator is the same class on row 103 and 104.